src/components/Team.js

```diff
@@ -0,0 +1,107 @@
+import Image from "next/image";
+import data from "public/data.json";
```

Comment:
using the preconfigured public path `~/`, so it would something like this `~/data.json` would be a better option.